interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -289,7 +289,7 @@
     <title><?php echo xlt('Rx Dispensed History'); ?></title>
     <head>
 
-        <?php Header::setupHeader([ 'opener', 'jquery-ui', 'jquery-ui-redmond', 'pure', 'jscolor' ]); ?>
+        <?php Header::setupHeader(['opener', 'pure', 'jscolor']); ?>
```

Comment:
Line 296 - Remove unneeded inline styling here. Things that should be removed are the table style, all input styles, right and left classes, bold class, and underline class

Reply:
Also do the same on line 700